syncopy/datatype/base_data.py

```diff
@@ -1685,6 +1685,9 @@ def trialdefinition(self, data):
                         trlTime = data._get_time([trlno], toilim=[-np.inf, np.inf])[0]
                         if isinstance(trlTime, list):
                             stop = np.max(trlTime)
+                            # Avoid creating empty arrays for "static" `SpectralData` objects
```

Comment:
so this adds more or less a bogus time-axis for trial-stacking of say Fourier-Spectra?

Reply:
Exactly, but only "static" (i.e., not time-frequency) spectra, which are one sample "long" which was not quantified correctly.